modyn/trainer_server/internal/trainer/pytorch_trainer.py

```diff
@@ -226,12 +257,25 @@ def connect_to_selector(self, selector_address: str) -> SelectorStub:
         return SelectorStub(selector_channel)
 
     def instantiate_downsampler(
-        self, strategy_name: str, params_from_selector: dict, per_sample_loss: Any
+        self, strategy_name: str, downsampler_config: dict, per_sample_loss: Any
```

Comment:
Could you add a comment here what exactly `per_sample_loss` is? I forgot but the type `Any` isn't super helpful. I am not sure what this is used for currently

Reply:
It was already used elsewhere and I lazily copied the typing.  Now it is torch.nn.modules.loss